lib/v2/screens/profile_screens/citizenship/components/citizenship_progress_item.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:step_progress_indicator/step_progress_indicator.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+class CitizenshipProgressItem extends StatelessWidget {
```

Comment:
these are used in many places of the app, not just citizens. 
Please make generic and move to a more generic place.

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/8041901/110878927-17c88300-82aa-11eb-9639-f194b378bc7d.png)

Reply:
You're right 👍